src/energy_comms/transform/epa.py

```diff
@@ -0,0 +1,29 @@
+import pandas as pd
+
+
+def transform(df):
+
+    # from HUD: https://www.huduser.gov/portal/datasets/usps_crosswalk.html
+
+    zip_code_crosswalk = pd.read_excel(
+        "/Users/mcastillo/Documents/GitHub/rmi-energy-communities/src/energy_comms/extract/ZIP_COUNTY_122021.xlsx",
```

Comment:
This file is very large so ideally it wouldn't live in the repo. HUD does have a zip code API that we can use, especially for automatic updates, but for now, this can be replaced with:

```
pd.read_excel("https://www.huduser.gov/portal/datasets/usps/ZIP_COUNTY_122021.xlsx", dtype={"zip": "string", "county": "string"})
```
Then we can remove this spreadsheet from the repo.

Also, I included the string dtypes for zip and county in the `pd.read_excel` call because else they will be read in as ints and you'll need to pad them with zeros. This way the zeros are read in from the sheet and you don't need to mess with zip and county so that they're both five digits.

Reply:
thanks - updated it with this link and kept the assigned dtypes!